lab4/src/Search.java

```diff
@@ -0,0 +1,24 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Search {
+
+    public static void main(String[] args) throws FileNotFoundException {
```

Comment:
you can consider using a try-catch block instead of throws since in this solution you never actually handle the exception to be thrown